### DISCUSSION TITLE
Gatsby Develop Segmentation fault

### DISCUSSION THREAD

**Discussion by luci4n** [gatsbyjs/gatsby#35933] [Help]
Hi! Can anyone help me fix the following error I get when runnig gastby develop:

$ gatsby develop
success compile gatsby files - 3.670s
success load gatsby config - 0.046s
success load plugins - 1.803s
success onPreInit - 0.008s
success initialize cache - 0.106s
success copy gatsby files - 4.748s
success Compiling Gatsby Functions - 0.383s
success onPreBootstrap - 1.465s
success createSchemaCustomization - 0.016s
success Checking for changed pages - 0.001s
success source and transform nodes - 1.433s
success building schema - 0.354s
success createPages - 0.054s
success createPagesStatefully - 0.134s
info Total nodes: 167, SitePage nodes: 41 (use --verbose for breakdown)
success Checking for changed pages - 0.000s
success write out redirect data - 0.003s
success Build manifest and related icons - 0.110s
success onPostBootstrap - 0.116s
info bootstrap finished - 23.359s
success onPreExtractQueries - 0.001s
success extract queries from components - 12.486s
success write out requires - 0.035s
warn [gatsby-transformer-sharp] The "fixed" and "fluid" resolvers are now
deprecated. Switch to "gatsby-plugin-image" for better performance and a simpler
 API. See https://gatsby.dev/migrate-images to learn how.
success run static queries - 0.086s - 2/2 23.21/s

Segmentation fault

**Comment by Jim-Horn** [comment]
I ran into a segmentation fault error when running one of the examples. I'd installed it using `yarn`, so tried again using `npm i` and it worked properly. I wonder if the two are related
